index.html

```diff
@@ -20,6 +20,13 @@ <h3 class="my-logo">My logo</h3>
                <img src="images/Enabled.png" alt="normal button">
            </div>
         </nav>
+        <nav class="nav-links nav-links1">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Kindly check your font color for your nav items, it differs from the design given, also some background images are missing, check the comparison below for better understanding;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/171600334-c5028a3d-3409-4c49-b1ec-9852a6c171fe.png) |  ![image](https://user-images.githubusercontent.com/61408860/171600394-c0982bd1-d6ed-43d4-a615-8e5974961741.png)